src/history/test/HistoryTests.cpp

```diff
@@ -818,22 +743,12 @@ TEST_CASE("Catchup recent", "[history][catchuprecent]")
 TEST_CASE("Catchup manual", "[history][catchupmanual]")
```

Comment:
While we are at it: let's not use `std::rand` as it has limited randomness

Reply:
I don't think we need true randomness in that part of the code. Also std::rand is used in some other tests. I think it would be better to replace them with true randomness at once in different PR.

Reply:
This can be done in a different PR (happy to look into it when I get a chance), but I disagree that the tests don't need it. Aside from bad randomness, when the default seed is used, rand produces the same sequence every time, and I thought the whole point was that we test different scenarios each time?